devicetypes/vision-kuowei/vision-in-wall-2relays-switch.src/vision-in-wall-2relays-switch.groovy

```diff
@@ -0,0 +1,281 @@
+/**
+ *  Vision In-Wall 2Relays Switch (Models: ZL7435xx-5)
+ * 
+ *  Author:
+ *    Lan, Kuo Wei
+ *
+ *	Product Link:
+ *    http://www.visionsecurity.com.tw/index.php?option=product&lang=en&task=pageinfo&id=335&belongid=334&index=0
+*/
+metadata {
+	definition(
+		name: "Vision In-Wall 2Relays Switch", 
+		namespace: "Vision_KuoWei", 
+		author: "Lan, Kuo Wei", 
+		ocfDeviceType: "oic.d.switch", 
+		deviceTypeId: "Switch",
+		mnmn: "0ALw", 
+		vid: "812dd85f-ae1e-382e-9289-15cbc7c7fd6f"
+	) {
+		capability "Health Check"
+		capability "Refresh"
+		capability "Switch"
+		capability "Configuration"
+		// This DTH uses 2 switch endpoints. Parent DTH controls endpoint 1 so please use '1' at the end of deviceJoinName
+		// Child device (isComponent : false) representing endpoint 2 will substitute 1 with 2 for easier identification.
+		fingerprint manufacturer: "0109", prod: "2017", model: "171B", deviceJoinName: "Vision 2Relays Switch 1" //zw:Ls type:1001 mfr:0109 prod:2017 model:171B ver:16.11 zwv:4.38 lib:03 cc:98 sec:5E,72,86,85,59,70,5A,7A,60,8E,73,27,25 epc:2
+		fingerprint manufacturer: "0109", prod: "2017", model: "171C", deviceJoinName: "Vision 2Relays Switch 1" //zw:Ls type:1001 mfr:0109 prod:2017 model:171C ver:25.07 zwv:4.54 lib:03 cc:98 sec:5E,72,86,85,59,70,5A,7A,60,8E,73,27,25 epc:2
+	}
+
+	preferences {
+		parameterMap.each {
+			input (title: it.name, description: it.description, type: "paragraph", element: "paragraph")
+			switch(it.type) {
+				case "boolean":
+					input(type: "paragraph", element: "paragraph", description: "Option enabled: ${it.activeDescription}\n" + "Option disabled: ${it.inactiveDescription}")
+					input(name: it.key, type: "boolean", title: "Enable", defaultValue: it.defaultValue == it.activeOption, required: false)
+					break
+					
+				case "enum":
+					input(name: it.key, title: "Select", type: "enum", options: it.values, defaultValue: it.defaultValue, required: false)
+					break
+			}	
+		}
+	}
+}
+
+def installed() {
+	/* Push main states */
+	sendEvent(name: "healthStatus", value: "online")
+	sendEvent(name: "DeviceWatch-DeviceStatus", value: "online")
+	sendEvent(name: "DeviceWatch-Enroll", value: [protocol: "cloud", scheme:"untracked"].encodeAsJson(), displayed: false)
```

Comment:
if you want the device's health to be tracked automatically, you'll need to replace this with 

`sendEvent(name: "checkInterval", value: 60 * 60 * [HOURS] + 60 * [MINUTES], displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])`

otherwise your device will never be marked offline

Reply:
Already removed that, bercause "checkInterval" already setting in configure().